pkgs/by-name/je/jet-pilot/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  fetchurl,
+  appimageTools,
+}:
+
+appimageTools.wrapType2 rec {
+  pname = "jet-pilot";
+  version = "1.28.2";
+
+  src = fetchurl {
+    url = "https://github.com/unxsist/jet-pilot/releases/download/v${version}/jet-pilot_${version}_amd64.AppImage";
+    hash = "sha256-G8HsSyxZUkdOR3tW5E1/yKo3ZJ6CKTnv2RzT+fnENA0=";
```

```suggestion
  version = "1.31.0";

  src = fetchurl {
    url = "https://github.com/unxsist/jet-pilot/releases/download/v${version}/jet-pilot_${version}_amd64.AppImage";
    hash = "sha256-jhy8BfCZ0Z4T3lGrxTmSGw+fSDVzTIuvyJftpwMcYo0=";
```
1.31 was released since my last review.
I will merge after you apply the update :)